packages/components/src/components/primitives/Chat/Conversation/Message/Message.tsx

```diff
@@ -47,10 +49,16 @@ export const Message = ({
   time,
   mediaFiles = [],
   onMediaClick,
+  onLoadMedia,
 }: Props) => {
+  const [isMediaLoaded, loadMedia] = useState(false);
```

Comment:
`loadMedia` suggest that the method loads the media, not that it set the value of this flag

Reply:
Renamed to `setMediaLoaded`